mllib/src/main/scala/org/apache/spark/ml/tree/Split.scala

```diff
@@ -85,7 +85,7 @@ final class CategoricalSplit private[ml] (
   /**
    * If true, then "categories" is the set of categories for splitting to the left, and vice versa.
    */
-  private val isLeft: Boolean = _leftCategories.length <= numCategories / 2
+  private[ml] val isLeft: Boolean = _leftCategories.length <= numCategories / 2
 
   /** Set of categories determining the splitting rule, along with [[isLeft]]. */
   private val categories: Set[Double] = {
```

Comment:
Should we maybe do this in a second change? It seems like if there aren't that many categories for one direction then this would maybe not be a great change. What are your thoughts?